projects/packages/sync/src/class-replicastore.php

```diff
@@ -1191,6 +1199,35 @@ public function checksum_all() {
 			'term_relationships' => $this->summarize_checksum_histogram( $term_relationships_checksum ),
 			'term_taxonomy'      => $this->summarize_checksum_histogram( $term_taxonomy_checksum ),
 		);
+
+		/**
+		 * WooCommerce tables
+		 */
+		if ( static::$force_woocommerce_support || class_exists( 'WooCommerce' ) ) {
```

Comment:
@bisko can we change this to a filter?  'jetpack_sync_support_woocommerce' or similar. This way we aren't relying on change a variable in the class but can add a filter for testing that will keep us from accidentally committing true.

Reply:
The variable is only overriden in the WordPress.com class that extends this one. The variable should always be `false` in the Jetpack Context and `true` in the WordPress.com context.